client/vmmVSwitchPolicyCont_service.go

```diff
@@ -0,0 +1,331 @@
+	dn := fmt.Sprintf("uni/vmmp-%s/dom-%s/vswitchpolcont", provider_profile_vendor, vmm_domain)
+	cont, err := sm.Get(dn)
+	if err != nil {
+		return nil, err
+	}
+
+	vmmVSwitchPolicyCont := models.VSwitchPolicyGroupFromContainer(cont)
+	return vmmVSwitchPolicyCont, nil
+}
+
+func (sm *ServiceManager) DeleteVSwitchPolicyGroup(vmm_domain string, provider_profile_vendor string) error {
+	dn := fmt.Sprintf("uni/vmmp-%s/dom-%s/vswitchpolcont", provider_profile_vendor, vmm_domain)
+	return sm.DeleteByDn(dn, models.VmmvswitchpolicycontClassName)
+}
+
+func (sm *ServiceManager) UpdateVSwitchPolicyGroup(vmm_domain string, provider_profile_vendor string, description string, vmmVSwitchPolicyContattr models.VSwitchPolicyGroupAttributes) (*models.VSwitchPolicyGroup, error) {
+	rn := fmt.Sprintf("vswitchpolcont")
+	parentDn := fmt.Sprintf("uni/vmmp-%s/dom-%s", provider_profile_vendor, vmm_domain)
+	vmmVSwitchPolicyCont := models.NewVSwitchPolicyGroup(rn, parentDn, description, vmmVSwitchPolicyContattr)
+
+	vmmVSwitchPolicyCont.Status = "modified"
+	err := sm.Save(vmmVSwitchPolicyCont)
+	return vmmVSwitchPolicyCont, err
+
+}
+
+func (sm *ServiceManager) ListVSwitchPolicyGroup(vmm_domain string, provider_profile_vendor string) ([]*models.VSwitchPolicyGroup, error) {
+
+	baseurlStr := "/api/node/class"
+	dnUrl := fmt.Sprintf("%s/uni/vmmp-%s/dom-%s/vmmVSwitchPolicyCont.json", baseurlStr, provider_profile_vendor, vmm_domain)
+
+	cont, err := sm.GetViaURL(dnUrl)
+	list := models.VSwitchPolicyGroupListFromContainer(cont)
+
+	return list, err
+}
+
+func (sm *ServiceManager) CreateRelationvmmRsVswitchExporterPol(parentDn, tDn, activeFlowTimeout, idleFlowTimeout, samplingRate string) (string, error) {
+	dn := fmt.Sprintf("%s/rsvswitchExporterPol-[%s]", parentDn, tDn)
+	containerJSON := []byte(fmt.Sprintf(`{
+		"%s": {
+			"attributes": {
+				"dn": "%s",
+				"activeFlowTimeOut": "%s",
+				"idleFlowTimeOut": "%s",
+				"samplingRate": "%s"
+			}
+		}
+	}`, "vmmRsVswitchExporterPol", dn, activeFlowTimeout, idleFlowTimeout, samplingRate))
+
+	jsonPayload, err := container.ParseJSON(containerJSON)
+	if err != nil {
+		return dn, err
+	}
+
+	req, err := sm.client.MakeRestRequest("POST", fmt.Sprintf("%s.json", sm.MOURL), jsonPayload, true)
+	if err != nil {
+		return dn, err
+	}
+
+	cont, _, err := sm.client.Do(req)
+	if err != nil {
+		return dn, err
+	}
+	fmt.Printf("%+v", cont)
+
+	return dn, nil
+}
+
+func (sm *ServiceManager) DeleteRelationvmmRsVswitchExporterPol(parentDn, tDn string) error {
+	dn := fmt.Sprintf("%s/rsvswitchExporterPol-[%s]", parentDn, tDn)
+	return sm.DeleteByDn(dn, "vmmRsVswitchExporterPol")
+}
+
+func (sm *ServiceManager) CreateRelationvmmRsVswitchOverrideFwPol(parentDn, tDn string) error {
+	dn := fmt.Sprintf("%s/rsvswitchOverrideFwPol", parentDn)
+	containerJSON := []byte(fmt.Sprintf(`{
+		"%s": {
+			"attributes": {
+				"dn": "%s",
+				"tDn": "%s"
```

Comment:
We usually add "annotation" parameter in the attributes payload with value as "orchestrator:terraform" if relation class support annotation.